src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -1988,6 +1996,13 @@ image::upload-catalog-objects.png[align=left]
 
 WARNING: In case of uploading an archive, all the objects in the zip must have the same Kind and Content Type.
 
+==== Create a zip archive with Catalog objects
+When the user creates a zip archive for uploading objects to Catalog, the hidden files should be excluded from the archive.
+
+===== Create a zip archive with Catalog objects in Mac OS
+First navigate to the folder with the objects that you want to archive. You can use next zip command in Terminal to archive the files without the `.DS_Store`, `__MACOSX` and other `.* files`:
+
+    zip -r ArchiveForCatalog.zip . -x ".*" -x "__MACOSX"
```

Comment:
not sure about the command. For my case, this command works:
zip -r ArchiveForCatalog.zip . -x *.DS_Store

I think that you have to put -x *.DS_Store

Reply:
yes, of course if you exclude only .DS_Store file it will work for mac (with -x *.DS_Store)
But I want to exclude also other hidden `.* files` if there are any.
Do you see the point?

Reply:
I see what do you mean but unfortunately this command zip -r ArchiveForCatalog.zip . -x ".*" does not exclude .DS_Store file.